package/thingino-esphome/thingino-esphome.mk

```diff
@@ -0,0 +1,81 @@
+################################################################################
+#
+# thingino-esphome
+#
+################################################################################
+
+THINGINO_ESPHOME_VERSION = 2e140651a0594e6d4289cdfde85799c25606b671
+THINGINO_ESPHOME_SITE = https://github.com/yinzara/esphome-linux
+THINGINO_ESPHOME_SITE_METHOD = git
+THINGINO_ESPHOME_SITE_BRANCH = main
+THINGINO_ESPHOME_LICENSE = MIT
+THINGINO_ESPHOME_LICENSE_FILES = LICENSE
+
+# Base dependencies
+THINGINO_ESPHOME_DEPENDENCIES = host-meson host-pkgconf thingino-libcurl ingenic-audiodaemon
+
+# Add Bluetooth dependencies if Bluetooth is enabled
+ifeq ($(BR2_PACKAGE_THINGINO_BLUETOOTH),y)
+THINGINO_ESPHOME_DEPENDENCIES += dbus libglib2 bluez5_utils
+endif
+
+# Add TFLite Micro and microspeech features dependencies if wake word is enabled
```

Comment:
These 3 lines bled through from my next PR, you can ignore it though as they won't affect anything in this PR.